server/services/prisonGroupService.ts

```diff
@@ -0,0 +1,64 @@
+export const National: string = 'National' as const
+
+export interface PgdRegion {
+  name: string
+  code: string
+  prisons: Prison[]
+}
+
+export interface Prison {
+  name: string
+  code: string
+}
+
+const allPrisonGroups: PgdRegion[] = [
```

Comment:
I included these 4 groups because they are either
- The group from the design (WM)
- Have punctuation in the names (HESG - comma, WE - apostrophe, LTHS - ampersand) and I thought was good to test these